tests/config/deno.json

```diff
@@ -1,4 +1,5 @@
 {
+  "lock": false,
```

Comment:
Why do we need this one?

Reply:
Otherwise, [this file](https://github.com/denoland/deno/pull/24543/commits/45fa3e252c013daf0f310a6ad13563dc584dbfc6#diff-51b5255abda12ef4c9fb2ca9f070ce1566f73ec53dc7ed7b0bc0e97d3ae4a346) is produced. Happy to revert.